aind-metadata-service-server/src/aind_metadata_service_server/mappers/procedures.py

```diff
@@ -266,10 +276,43 @@ def map_responses_to_aind_procedures(
                 f"from NSB2019 for {subject_id}"
             )
 
+        if self.nsb_2023:
+            nsb_2023_surgeries = (
+                self.map_sharepoint_response_to_aind_surgeries(
+                    response=self.nsb_2023,
+                    mapper_cls=MappedNSB2023,
+                )
+            )
+            subject_procedures.extend(nsb_2023_surgeries)
+            logging.info(
+                f"Found {len(nsb_2023_surgeries)} surgeries "
+                f"from NSB2023 for {subject_id}"
+            )
+
+        if self.nsb_present:
+            nsb_present_surgeries = (
+                self.map_sharepoint_response_to_aind_surgeries(
+                    response=self.nsb_present,
+                    mapper_cls=MappedNSB2023,
+                )
+            )
+            subject_procedures.extend(nsb_present_surgeries)
+            logging.info(
+                f"Found {len(nsb_present_surgeries)} surgeries "
+                f"from NSB Present for {subject_id}"
+            )
+
         if not subject_procedures and not specimen_procedures:
             return None
-        return Procedures(
-            subject_id=subject_id,
-            subject_procedures=subject_procedures,
-            specimen_procedures=specimen_procedures,
-        )
+        try:
+            return Procedures(
+                subject_id=subject_id,
+                subject_procedures=subject_procedures,
+                specimen_procedures=specimen_procedures,
+            )
+        except (ValidationError, CoordinateSystemException):
```

Comment:
A bug in aind-data-schema v2 is causing this exception to be raised. @dbirman has a fix in dev that will wrap it in ValidationError, but until that's released we need to catch it